panic-probe/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "panic-probe"
+version = "0.1.0"
+authors = ["The Knurling Authors"]
+edition = "2018"
+description = "Panic handler that exits `probe-run` with an error code"
+
+[dependencies]
+cortex-m = "0.6.3"
+cortex-m-rt = "0.6.12"
+
+[dependencies.rtt-target]
+version = "0.2.2"
+optional = true
+
+[dependencies.defmt]
+git = "https://github.com/knurling-rs/defmt.git"
+branch = "main"
+optional = true
+
+[features]
+# Print the panic message using `rtt-target`.
+print-rtt = ["rtt-target"]
+# Print the panic message using `defmt`.
+print-defmt = ["defmt", "defmt-error"]
+
+defmt-error = [] # internal feature, do not use
```

Comment:
where is this Cargo feature used from?

Reply:
The `print-defmt` feature enables it

Reply:
ah, right this is from the `defmt` log macro; didn't explicitly see it as `cfg(feature = "defmt-error")` when I searched through diff. But that makes sense.